examples/language-modeling/README.md

```diff
@@ -95,6 +95,57 @@ python ../gaudi_spawn.py \
 This takes about 4 minutes to train on 8 HPUs. It reaches
 a perplexity of 21.7968 once fine-tuned on the dataset.
 
+## GPT-J-6B and causal language modeling
+
+The following command triggers fine-tuning of GPT-J-6b model on WikiText-2 with Deepspeed ZeRO-2. Fine tuning on 8 HPU cards takes around 6 mins with batchsize of 4.
+It reaches a perplexity of 14.011 once finetuned with the dataset.
+### Multi-card Training with Deepspeed
+
+```bash
+python ../gaudi_spawn.py \
+    --world_size 8 --use_deepspeed run_clm.py \
+    --model_name_or_path EleutherAI/gpt-j-6b \
+    --dataset_name wikitext \
+    --dataset_config_name wikitext-2-raw-v1 \
+    --per_device_train_batch_size 4 \
+    --per_device_eval_batch_size 4 \
+    --do_train \
+    --do_eval \
+    --output_dir /tmp/test-clm-xl-1 \
+    --gaudi_config_name Habana/gpt2 \
+    --use_habana \
+    --use_lazy_mode \
+    --gradient_checkpointing \
+    --use_hpu_graphs_for_inference \
+    --throughput_warmup_steps 2 \
+    --deepspeed path_to_my_deepspeed_config
+```
+## GPT-J-6B and causal language modeling
+
+The following command triggers fine-tuning of  GPT-NEOX-20b model on WikiText-2 with Deepspeed ZeRO-2. Fine tuning on 16 HPU cards takes around 9 mins with batch size of 2
```

Comment:
```suggestion
The following command triggers fine-tuning of  GPT-NeoX-20B model on WikiText-2 with Deepspeed ZeRO-2. Fine tuning on 16 HPU cards takes (2 Gaudi2 instances) around 9 mins with batch size of 32 (per device is 2).
```

Reply:
@ZhaiFeiyue : Addressed the review comments